instrumentation/github.com/go-sql-driver/mysql/splunkmysql/example_test.go

```diff
@@ -0,0 +1,32 @@
+// Copyright Splunk Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package splunkmysql_test
+
+import (
+	"time"
+
+	"github.com/signalfx/splunk-otel-go/instrumentation/database/sql/splunksql"
+	_ "github.com/signalfx/splunk-otel-go/instrumentation/github.com/go-sql-driver/mysql/splunkmysql"
+)
+
+func Example() {
+	db, err := splunksql.Open("mysql", "user:password@/dbname")
+	if err != nil {
+		panic(err)
+	}
+	db.SetConnMaxLifetime(time.Minute * 3)
+	db.SetMaxOpenConns(10)
+	db.SetMaxIdleConns(10)
```

Comment:
how about putting `db.ExecContext(ctx, query)` instead?

it may be not obvious that passing `ctx` e.g. from the request is important

I think we could also add the default distro setup